src/tests/ftest/telemetry/telemetry_pool_metrics.py

```diff
@@ -22,163 +20,155 @@ class TelemetryPoolMetrics(IorTestBase, TestWithTelemetry):
-            DAOS engine IO telemetry basic metrics infrastructure.
+            Create a file of 500MiB thanks to ior with a transfer size of 1MiB to verify the DAOS
+            engine IO telemetry basic metrics infrastructure.
         Steps:
             Create Pool
             Create Container
-            Generate deterministic workload. Using ior to write 512M
-            of data with 1M chunk size and 1M transfer size.
+            Generate deterministic workload. Using ior to write 512MiB
+            of data with 1MiB chunk size and 1MiB transfer size.
             Use telemetry command to get values of 4 parameters
             "engine_pool_ops_fetch", "engine_pool_ops_update",
             "engine_pool_xferred_fetch", "engine_pool_xferred_update"
             for all targets.
             Verify the sum of all parameter metrics matches the workload.
-            Do this with RF=0 and RF2(but RP_2GX).
+            Do this with RF=0 and RF2(but RP_3GX).
             For RF=0 the sum should be exactly equal to the expected workload
-            and for RF=2 (with RP_2GX) sum should be double the size of
+            and for RF=2 (with RP_3GX) sum should triple the size of
             workload for write but same for read.
 
         :avocado: tags=all,daily_regression
         :avocado: tags=vm
         :avocado: tags=telemetry
         :avocado: tags=telemetry_pool_metrics,test_telemetry_pool_metrics
         """
-        # test parameters
-        self.threshold_percent = self.params.get("threshold_percent",
-                                                 "/run/telemetry_metrics/*")
-        metric_list = ["engine_pool_ops_fetch",
-                       "engine_pool_ops_update",
-                       "engine_pool_xferred_fetch",
-                       "engine_pool_xferred_update"]
-        metrics_data = {}
+
         # create pool and container
-        idx = 0
         self.add_pool(connect=False)
         self.pool.set_property("reclaim", "disabled")
         self.add_container(pool=self.pool)
+
         # collect first set of pool metric data before read/write
-        metrics_data[idx] = self.telemetry.get_pool_metrics(metric_list)
-        idx += 1
+        metric_names = [
+            "engine_pool_ops_fetch",
+            "engine_pool_ops_update",
+            "engine_pool_xferred_fetch",
+            "engine_pool_xferred_update",
+            "engine_pool_resent"
+        ]
+        metrics_data = []
+        metrics_data.append(self.telemetry.get_pool_metrics(metric_names))
 
         # Run ior command.
         try:
             self.update_ior_cmd_with_pool(False)
-            self.ior_cmd.dfs_oclass.update(self.container.oclass.value)
+            self.ior_cmd.dfs_oclass.update(self.dfs_oclass)
+            self.ior_cmd.transfer_size.update(self.ior_transfer_size)
+            self.ior_cmd.dfs_chunk.update(self.ior_transfer_size)
+            self.ior_cmd.block_size.update(self.ior_block_size)
```

Comment:
If `transfer_size` is always the same from the config, we could remove `self.ior_transfer_size` and just use `self.ior_cmd.transfer_size.value` when needed, since it is set automatically when `self.ior_cmd` is setup by the framework.
Similar for `dfs_chunk` and `block_size`.

I.e. by the time we get here, `self.ior_cmd.transfer_size.value`, `self.ior_cmd.dfs_chunk.value`, and `self.ior_cmd.block_size.value` will already match what is in the yaml.

Reply:
- [ ] Remove useless attributes.